libnet/client.c

```diff
@@ -0,0 +1,137 @@
+#include "client.h"
+
+#define _GNU_SOURCE 201112L
+
+#include <arpa/inet.h>
+#include <fdebug.h>
+#include <netdb.h>
+#include <pthread.h>
+#include <sys/select.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+
+#include "common.h"
+
+client_info server[1];
+
+inline unsigned get_client_id(client_info *client) {
```

Comment:
This function is used in common part. For server build it returns appropriate client id and for client build returns 0 as it's the only possible connected client (code treats server as one and only connected client)